README.md

```diff
@@ -171,42 +171,41 @@ ember exam --split=3 --partition=3
 $ ember exam --load-balance=<num>
 ```
 
-`load-balance` allows you to specify a number of browser(s) to load balance test files against. It will order the test files by test types with test load balancing. If you specify a value for load-balance of 3, then three browser instances will be created and the output looks something like:
+`load-balance` allows you to specify a number of browser(s) to load test files against. It will order the test files by test types with test load balancing. For example, you specify load-balance equals 3, then three browser instances will be created and the output will look something like:
```

Comment:
The 'a number' should be 'the number'.
The 'by test types with test load balancing' should be 'by test type before doing load balancing'.
The 'For example, you' should be 'For example, if you'